examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
apply_proj should be smart enough to know which projectors have been
applied and apply if necessary. Sounds like maybe it's failing in that job.
Or perhaps the projectors that get added have the active flag True? They
probably shouldn't since they haven't been applied to the data. I thought
apply_proj looped through the list of projectors to see if the projectors
had been applied, and did nothing if they were all already marked active.

The log messages should be informative but I'm on mobile so can't check
now.